docs/http.md

```diff
@@ -311,7 +311,7 @@ All other properties might be undefined.
 
 ### Trip service
 
-The trip plugin solves the Traveling Salesman Problem using a greedy heuristic (farthest-insertion algorithm).
+The trip plugin solves the Traveling Salesman Problem using a greedy heuristic (farthest-insertion algorithm) for more than ten nodes and uses brute force for 10 nodes or less.
```

Comment:
That's not true: < 10 is brute force

https://github.com/Project-OSRM/osrm-backend/blob/b3ef27d104787af8c82ee11d51769a2164bce9b2/src/engine/plugins/trip.cpp#L209